conf/apache/nebnews-redirects.conf

```diff
@@ -0,0 +1,4 @@
+# https://github.com/CDRH/open-oni_nebraska_theme/issues/80
+RedirectMatch 301 ^/lccn/sn(201627020[0-2].*) /lccn/$1
```

Comment:
🎊

Reply:
I had tried to combine both with a non-capturing group, but it didn't seem to match so I split it up. I guess `RedirectMatch` may not understand `(?:[0-2]|[4-7])` unless I had a typo or something. Either way it's short overall for the amount of URLs it covers :smile: